backend/inspirehep/records/marshmallow/literature/ui.py

```diff
@@ -187,6 +192,26 @@ def add_ads_links_for_arxiv_papers(self, data):
 
         return data
 
+    def is_document_arxiv(self, document):
+        return document.get("source") == "arxiv"
+
+    def is_document_hidden(self, document):
+        return document.get("hidden", False)
+
+    def get_citation_pdf_urls(self, data):
+        urls = []
+        for document in data.get("documents", []):
+            url = document.get("url")
+            if (
+                url
+                and not self.is_document_hidden(document)
+                and self.is_document_arxiv(document)
```

Comment:
this logic is wrong, we should return all documents that are not `hidden` (we don't need the `is_document_arxiv` check)

Reply:
Ah ok I misunderstood requirement. :+1: